web-scraper/playbooks/playbooks.json

```diff
@@ -0,0 +1,200 @@
1hpKftXmL1iLrtWG8GEJAmhlFm2ZQ9dePJEkeR7WC7FWZiSCcPaNtjrD+H4NBSUaR3wPfEeh+lThjcmqHtUAQkkiFsv1XLGCOi7UrtGwZXEQlDbnaZRgt6s8umO6MZLll3R9v331RH9PS0doMLDoRCSoP1Pc5VBECrnDhSBG9ZrTpAsLy54vL6CmstkgKCpOlOCC4iVlfUtqbr18xGM9J0n6otSNM77O1+gazYYeg2DLalqi/IsneYz98kikafWYOeMMkkZbtAqtcZFY8o60vsIAhhDyGg7z6kqh4T3D5K3ca6NVV1ShrNESHQdscokRBsjRID42SEVgKtRmgJtr/iavMC4QsEksFeIQhokZGaKbEcMYSSzq6IREIqE6bRPjvxLqvunNPq5yy7p+je1zRuTWuXgCGNdzAy/0y2uotMFZNRxMG+ZTF7QJ68Qlk1rFdX9N1AP9QMQ4zWBXXdQ3iJD4q6fsxgA5v1PnDNcvmEYTBYtwPhgLKqKatLqvIFnoKuy2maGmtbYu2p5QolblM1U/r+y7TNira7oshfRaucgUN8+CmrzRnTyYI8niP9gPSOVE9pVErfv0Tg6fuYeDpwsDtDEuOdIUsDTXfEavUEJWYQPM7XBD0nY0qq5ySkVMMJ5XBKLCJiodEqZR7t4e3AZfsp+qZ5TGMvsKElUtNtOVc7AhItc4yGLLE4W3Ny5Lg6u2FVXnF185TerrE+IEUE4pzBvsBE4N2E86tTYEE/VGxqT1WNUfo+IuwgxYK2fUFde6S6QxIdEqldqv6I2Bxs46V4xWLxBcryHlVdQbim7x6Rxg9QMkKEAs2Y5c3fElzLaHxAEk2YFvvEWtH3l5TBbz02/ZKqEtSjhkmRoWVKFgvyzHOzOaEdbgheYH2JCxWpHGFlwjgaI0XHZf2Ym86jgV4mKCCWCTvJfXQznONCQ6RGRHJMokYU8Q5SxpghRpo1eeaRoScMcwafUlWWTbkEGZOn9xBSUlaf0NmSLLtF7zustQhuEOGc4N8mMEapEUk0ItEZ+IHgHCbeR4kY7wba9iUiSASKtsnouxH4QF2f03U3dO0LVjeayMyxrtxuKeSYunlGm95HBoXIdpnnu1SbO8gg6O32dNO1Q10f9wAABWNJREFUmpOzM+qmZzF+QAiCLNHkmWS9WWLdsJVrhIpSxkjpMPKQ3IzpkwlNf8LKWTIxwYd+2+k2Gl3oXRCK1Cwwasw4OSCJx3S2pbUrlMzABayLyLIRAUXdXG6DhuguWkwZ3A1te8HgwPW36LsbfIgRBLxfIMKraNEydGdcXX5CVV6wWf8CZ5fo5BZ4S9O9oC5/jEKh5X3qOqJvc4xOyaKtBHuIdhi6c7JoAcIQ6YQ8PaSunyKCJdIBoxzTfM7QPeClq4hVQmcjlBiD1ZxffkjfduTJDoFAHGuirkXpHh0cgz2hFzUbd8rQ7TJmgUkEUp+xsY+xOkNpgQsdtfJoJfT2sO0tQgUEnrq9Zt1d0LkNUYjo+xYTO5KpphmWKLVif3GPUf4AO3hW9RpEhdGvEOuvYYdTErNCqTGz0b9inN9nXZ9TNcd07fFWL6wLIhUzTvfxwbEezhGhQooGJSxDv8H2a1JtKNKACI5GaYK/RopnaOFRIpAlp7TtQ1zICeEVqmaNka+gZI1WGwZ3Q2IkRvUYmVI1EUfNhxRZTBzlhNAiVY/QFUWa0LcFdugJ+glNeEE/5CQyRZuAkyUlglSN0EqCl+hML/A4HB4ttyttZ0tCcBRmB6k0rb9gMcm4dZixaa4JZkESHWJURtmsaX1LkiQYsU+sI3otGKX3ieMFs1FOZEqa9hJ4TNuXwN2tPV0MWOcJfsDaj0E8J3iN4ATEiq69JjFThCi31RJxSZKcIfUjpIrQkSePA4Nz6PgnBHVF2T8Hcb21rsuGwBVCZigVo5Qji2a0TUTZHDH4EVpmKG3wPsEoQ5QntDUoM6CNpWt6mr5GOIsOI6Kwh3ELokgRgkU39pzBbxOMxBi60GNpUUogpWTwJU6+JB/vYpIlyt2QJJ7gl7TDira/YXCnKN2SmGuQHxFFK2YmJo4cWf4J2lhS+5zaPoK+RsgXKD9HKksITxHKofVLkqgjiV9SFB1d16DjGhWP0Qg8HrOxINdo7T/jUFK0eJMi3sGFFwh5TrCBzib4IcfaEikc+AFr19umfTSQ2IJ6SHCuQwpFbFK8zQhWoqPt32aMJstTUqNZrUraYY31HcJrgpshbAKqRvfqU4QJTIpD5kVPlkRYF9H2Dk/NsnpGFF2QZDmDWzHYDiUSkIIhWEJoUbInTR1ZeoYIDZECIw+Rqgd1hVQSoS5QMiaNFqAvCHaNkB0OjyCArDFxjDYxQWiEKBh6wdDnKJkz2BOCvN6eHPQOgoRhyHB1iveASglskNLhXMXgegZXEbxBKI2zDiksQtdoHdAuxdoaQYuSOVpk+EFjgySIJW1bIcWYWGXk0QJ8Qm3P2LhHBD8w5R4ySPTuTkae5dzZvce4mKNEzGbTsdyUlO0NTpyh5UBZNpTlMVVTYa1Hq4TBem6qEhsG4ihHoGm6M7wHnRh6WzJsChR7XN9ssM6SJRovDEGm27gpbHC2QwhBkU0o8pRIB2RIkGKECDO0zMG1KFoGH3BOopXBW0Nta4RskHT0w0CqEoI1CBcQDDgGXFAQFL53BFsy+GobkISY4AR2ULggEUhwOQiNdWtWjSU1ASML4pDhMZT+mBt+jh1W5GEfXeSavcWcrIgxMXR1w6ZeU3cbVptTyrokGlLObYV3DuvBMxBFAu80Q2dI0h3GWU439NjBopQg0jnt0GKbfTo7pa4tWndIBZIRShwilaMdXtIPFwiREOtdVDDYfjsoIHG2IijFYBus9YSwFSH6UNL7gB0meNZgL+ntCGckqUwJ3oPYbCfIWRAZwTv6rqT3FULVRDoHq7BOEPwWdnQSlNDgM3q7wQ43xEqgMCihyfSCajih8i8IvuL/AfwxIRQ0yHerAAAAAElFTkSuQmCC",
```

Comment:
can point to large_icon.png. The framework takes care of correctly updating it during connector install